microsoft/PowerToys
# Https scheme fix

## Summary of the Pull Request

**What is this about:**
In Powertoy Run some schemes were incorrectly being overwritten with HTTPS.  This change fixes this.

**What is include in the PR:** 
Should only be changes to 2 files

**How does someone test / validate:** 
Execute Powertoy Run, type mailto:test@example.com, you should see the correct scheme (mailto) in the drop down. 

## Quality Checklist

- [x] **Linked issue:** #12764
- [x] **Installer:** Added/updated and all pass (no changes to installer)
- [x] **Docs:** Added/ updated - should not be any changes required
